### PR TITLE
AC-455:Fixed App crashes while showing toast.

### PR DESCRIPTION
**Pull request by StonyMoon**
issue link: https://issues.openmrs.org/browse/AC-455
please review : )
I found some variable name such as: 
private View view;
Should I replace view with mView when fixing the issue?

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java [hunk 1]
```diff
@@ -54,7 +55,7 @@ public static PatientDetailsFragment newInstance() {
     @Override
     public void attachSnackbarToActivity() {
         Snackbar snackbar = Snackbar
-                .make(getActivity().findViewById(R.id.patientDashboardContentFrame), getString(R.string.snackbar_no_internet_connection), Snackbar.LENGTH_INDEFINITE);
+                .make(mPatientDashboardActivity.findViewById(R.id.patientDashboardContentFrame), getString(R.string.snackbar_no_internet_connection), Snackbar.LENGTH_INDEFINITE);
```
Comment (Sub6Resources) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java]: You're trying to access the `mPatientDashboardActivity` before it is instantiated.

Comment (StonyMoon) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java]: done. Please review.

Comment (Sub6Resources) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java]: You're correct. Looks good!

**Comment by shivtej1505** [comment]
The app is still crashing, attached log file on Jira.
@StonyMoon , you should use some descriptive variable name like `fragmentView` or `rootView`. `mView` doesn't describe anything.

**Comment by StonyMoon** [comment]
@shivtej1505 Thank you for your correction! Please review.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java [hunk 2]
```diff
@@ -99,14 +107,16 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
+            if (("M").equals(patient.getPerson().getGender())) {
+                ((TextView) rootView.findViewById(R.id.patientDetailsGender)).setText(getString(R.string.male));
+            } else {
+                ((TextView) rootView.findViewById(R.id.patientDetailsGender)).setText(getString(R.string.female));
+            }
         }
         ImageView patientImageView = (ImageView) rootView.findViewById(R.id.patientPhoto);
 
-        if (patient.getPerson().getPhoto() != null) {
+        if (patient.getPerson().getPhoto() != null ) {
```
Comment (shivtej1505) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/details/PatientDetailsFragment.java]: No need of this.

**Comment by dkayiwa** [comment]
@shivtej1505 you already have permission to merge the pull requests you approve for this project. Don't you? 😊

**Comment by StonyMoon** [comment]
@shivtej1505 Done. I have corrected the commit message.  :blush:

**Comment by shivtej1505** [comment]
@dkayiwa , I didn't realize that I can change the commit message while merging.

**Comment by dkayiwa** [comment]
@shivtej1505 you also have permission to close the corresponding JIRA ticket whenever you see it fit. Don't you?

**Comment by shivtej1505** [comment]
@dkayiwa , I know that. I usually close the JIRA tickets after two weeks. In this time other can confirm if there is any problem occurs by merging the PR.

**Comment by dkayiwa** [comment]
Great! 
I just wanted to make sure that you know of the empowerment that you have earned because of your awesome efforts! 👍

**Comment by shivtej1505** [comment]
Thanks @dkayiwa for helping and motivating me in every step :D

**Pull request merged**